pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -25,15 +25,15 @@ type mlnxNic struct {
 }
 
 const (
-	EthLinkType           = "ETH"
-	InfinibandLinkType    = "IB"
-	PreconfiguredLinkType = "Preconfigured"
-	UknownLinkType        = "Uknown"
-	TotalVfs              = "NUM_OF_VFS"
-	EnableSriov           = "SRIOV_EN"
-	LinkTypeP1            = "LINK_TYPE_P1"
-	LinkTypeP2            = "LINK_TYPE_P2"
-	MellanoxVendorId      = "15b3"
+	EthLinkType        = "ETH"
+	InfinibandLinkType = "IB"
```

Comment:
maybe move EthLinkType and InfinibandLinkType to utils/consts.go or similar
and use them in both mellanox plugin and utils.go (getLinkType())

Reply:
It was meant for mellanox plugin to not depend on the utils, the `getlinkType()` in utils is for Discovery and update the status for the NICs of the Node